### ISSUE TITLE
Tests failed with Python 3.9.1

### ISSUE THREAD

**Issue by yangfl** [micropython/micropython#6750]
https://github.com/micropython/micropython/blob/f7aafc0628f2008d015b32b0c0253a13f748d436/tests/extmod/urandom_basic.py#L25-L29

Python now accepts `random.getrandbits(0)`, it simply returns 0.

https://github.com/micropython/micropython/blob/f7aafc0628f2008d015b32b0c0253a13f748d436/tests/import/pkg7/subpkg1/subpkg2/mod3.py#L7-L11

Python now throws `ImportError` instead `ValueError`

```
Traceback (most recent call last):
  File "/.../micropython/tests/import/import_pkg7.py", line 2, in <module>
    import pkg7.subpkg1.subpkg2.mod3
  File "/.../micropython/tests/import/pkg7/subpkg1/subpkg2/mod3.py", line 9, in <module>
    from .... import mod1
ImportError: attempted relative import beyond top-level package
```

**Comment by dpgeorge** [comment]
> Python now throws `ImportError` instead `ValueError`

Do you know if there's a CPython PEP or bpo for this change?

**Comment by dpgeorge** [comment]
`getrandbits(0)` is fixed in c3199f56494429ba44f7e722597945dc91f3589c

**Comment by dpgeorge** [comment]
> > Python now throws `ImportError` instead `ValueError`
> 
> Do you know if there's a CPython PEP or bpo for this change?

It's here: https://bugs.python.org/issue37444

**Comment by dpgeorge** [comment]
See #7336 for a fix for the relative import error.

**Comment by dpgeorge** [comment]
Relative import error fixed in 53519e322a5a0bb395676cdaa132f5e82de22909
